.github/CHANGELOG.md

```diff
@@ -12,8 +12,34 @@
 
 ### Contributors
 
+---
+
+# Release 0.24.0
+
+### New features since last release
+
+* Added a cuquantum-backed finite shot sampling method. [(#21)](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/21).
+
+* Added more cuquantum-backed gates (IsingXX/YY/ZZ,MultiRz,SingleExcitation,...):  [(#28)](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/28).
+
+### Breaking changes
```

Comment:
I think the update to C++20 is a breaking change for those with older tooling. Worth a mention?